[0.2] Hotfix - Show delete discount

wychoong opened the pull request.

alecritson commented:
Thanks @wychoong Can you target the `0.2` branch please. :)

alecritson commented:
@wychoong Looks like the UI is a bit off, do you think this should appear in the left column rather than spanning across the whole page?

wychoong commented:
> @wychoong Looks like the UI is a bit off, do you think this should appear in the left column rather than spanning across the whole page?

<img width="1301" alt="image" src="https://user-images.githubusercontent.com/67364036/218244060-4cc66ee9-ea5e-46ca-9abc-e8a1b7edb0c0.png">

done. I also moved the Save button to bottom panel to align with other pages. and refactored the bottom panel to a separate component
